rpc/trace.go

```diff
@@ -0,0 +1,78 @@
+package rpc
+
+import (
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/juno/utils"
+	"github.com/NethermindEth/juno/vm"
+)
+
+type TransactionTrace struct {
+	// todo add the rest of types
```

Comment:
Why are we missing the rest?

Reply:
I've been thinking how to properly declare enum of different transaction types (choice between struct with same field + omitempty or private interface). I decided to use the latter.